tests/common/widgets/hits/index.ts

```diff
@@ -18,7 +19,20 @@ export function createHitsWidgetTests(
     document.body.innerHTML = '';
   });
 
-  describe('Hits widget common tests', () => {
+  describe('Hits widget common Insights tests', () => {
     createInsightsTests(setup, { act, skippedTests });
   });
 }
+
+export function createHitsWidgetOptionsTests(
+  setup: HitsWidgetSetup,
+  { act = fakeAct, skippedTests = {} }: TestOptions = {}
+) {
+  beforeEach(() => {
+    document.body.innerHTML = '';
+  });
+
+  describe('Hits widget common option tests', () => {
+    createOptionsTests(setup, { act, skippedTests });
+  });
+}
```

Comment:
```suggestion
    createInsightsTests(setup, { act, skippedTests });
    createOptionsTests(setup, { act, skippedTests });
  });
}
```

this works too right?